app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -53,55 +58,75 @@ export class ArticleCoverImage extends Component {
     });
   };
 
+  onDropImage = (event) => {
+    onDragExit(event);
+
+    if (event.dataTransfer.files.length > 1) {
```

Comment:
is there a reason for this limitation? I'm referring to Github's editor which allows multiple ones. Non blocking obvs, but I was wondering why we can't allow more than 1 and use promises to upload them concurrently. We could set an arbitrary max. 

I just uploaded 25 in thix box and it worked. The UX isn't great as it attaches the markdown in the position of the cursor which means that if you move the cursor during update it'll create invalid tags mashed with each other.

Reply:
From what I can tell right now, our backend is only equipped to receive one image in the POST request. We can definitely allow for multiple though. I'll revisit this in the cooldown period.